docs/4.0/content/reboot.md

```diff
@@ -127,38 +127,33 @@ All lists—`<ul>`, `<ol>`, and `<dl>`—have their `margin-top` removed and a `
 6. Faucibus porta lacus fringilla vel
 7. Aenean sit amet erat nunc
 8. Eget porttitor lorem
-{% endmarkdown %}
+{% endcapture %}
+{{ markdown | markdownify }}
 </div>
 
 For simpler styling, clear hierarchy, and better spacing, description lists have updated `margin`s. `<dd>`s reset `margin-left` to `0` and add `margin-bottom: .5rem`. `<dt>`s are **bolded**.
 
 <div class="bd-example">
-{% markdown %}
-<dl>
-  <dt>Description lists</dt>
-  <dd>A description list is perfect for defining terms.</dd>
-  <dt>Euismod</dt>
-  <dd>Vestibulum id ligula porta felis euismod semper eget lacinia odio sem.</dd>
-  <dd>Donec id elit non mi porta gravida at eget metus.</dd>
-  <dt>Malesuada porta</dt>
-  <dd>Etiam porta sem malesuada magna mollis euismod.</dd>
-</dl>
-{% endmarkdown %}
+  <dl>
+    <dt>Description lists</dt>
+    <dd>A description list is perfect for defining terms.</dd>
+    <dt>Euismod</dt>
+    <dd>Vestibulum id ligula porta felis euismod semper eget lacinia odio sem.</dd>
+    <dd>Donec id elit non mi porta gravida at eget metus.</dd>
+    <dt>Malesuada porta</dt>
+    <dd>Etiam porta sem malesuada magna mollis euismod.</dd>
+  </dl>
 </div>
 
 ## Preformatted text
 
 The `<pre>` element is reset to remove its `margin-top` and use `rem` units for its `margin-bottom`.
 
-<div class="bd-example">
-{% markdown %}
-<pre>
+{% highlight css %}
```

Comment:
Need to confirm the output doesn't change due to any of our custom docs CSS.

Reply:
| Before | After |
| --- | --- |
| <img width="576" alt="screen shot 2018-01-16 at 18 04 04" src="https://user-images.githubusercontent.com/26371/35001781-e1d1475c-fae7-11e7-96f7-e9ffd8c295ad.png"> | <img width="586" alt="screen shot 2018-01-16 at 18 04 17" src="https://user-images.githubusercontent.com/26371/35001795-ec68dd6a-fae7-11e7-972f-cefd05478094.png"> |

Reply:
@mdo: did you meant this to be just plain `pre` without highlighting?

Reply:
Yes, it should be without highlighting since it's only showing the `<pre>` element.